src/util/loanUtils.js

```diff
@@ -93,21 +93,60 @@ export function build5DollarsPriceArray(amountLeft) {
 	return priceArray;
 }
 
-export function getDropdownPriceArray(unreservedAmount, minAmount, enableFiveDollarsNotes, inPfp = false) {
+function buildHugePriceArray(amountLeft) {
+	const minAmount = 100;
+	const limitAmount = amountLeft > 1000 ? 1000 : amountLeft;
+	const N = limitAmount / minAmount;
```

Comment:
`N` should probably be renamed to something more consistent with other variable names.

Reply:
Agreed, but should apply for all cases we have this.